src/app/store/chains/stx-chain.actions.ts

```diff
@@ -13,25 +13,30 @@ export function initializeIndexZeroAccount(): AppThunk {
   return async (dispatch, getState) => {
```

Comment:
Doesn't look like these need to be async. Noop, but the new eslint setup warns about these.

Reply:
Let's migrate asap to be on the same eslint config